circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   xcode:
-    version: 8.2
+    version: 8.3
```

Comment:
Spotlight is just completely broken on CI, nothing I try with `mdfind` works.

Maybe we should change our rakelib so that, when we can't find Xcode, we default to `/Applications/Xcode.app`. If we can't find that, then we throw an error.

The possible alternatives aren't really viable: system_profiler depends on spotlight, and pkgutil is too slow.